torch/_dynamo/skipfiles.py

```diff
@@ -160,7 +160,15 @@ def add(import_name: str):
     if isinstance(import_name, types.ModuleType):
         return add(import_name.__name__)
     assert isinstance(import_name, str)
-    module_spec = importlib.util.find_spec(import_name)
+    try:
+        module_spec = importlib.util.find_spec(import_name)
+    except AttributeError as e:
+        # handle fbgemm conflict caused by import torchrec.
+        if "'fbgemm' object has no attribute" in str(e):
```

Comment:
This is not amazing, what is the actual exception here? When does this come up? Why do we ignore this exception?